src/bundle/Resources/public/js/CKEditor/embed/image/embed-image-editing.js

```diff
@@ -170,7 +170,7 @@ class IbexaEmbedImageEditing extends Plugin {
                 const link = viewElement.getChild(1);
                 const modelElement = upcastWriter.createElement('embedImage', { contentId, size });
 
-                if (link) {
+                if (link instanceof Element) {
```

Comment:
I think we should be more strict in this check
```suggestion
                if (link?.is('element', 'a')) {
```